package.json

```diff
@@ -44,10 +44,12 @@
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-stage-2": "^6.24.1",
     "babel-preset-stage-3": "^6.24.1",
+    "bignumber.js": "^8.1.1",
     "bip39": "^2.5.0",
     "chai": "4.2.0",
     "chai-as-promised": "^7.1.1",
     "chai-bignumber": "3.0.0",
+    "chai-bn": "^0.1.1",
```

Comment:
should remove not used or replacement packages like `chai-bignumber`.

Reply:
This is required because truffle is returning BNs

Reply:
yeah but we don't need chai-bignumber anymore!?

Reply:
Removed!

Reply:
great!